### ISSUE TITLE
Your address is public

### ISSUE THREAD

**Issue by gobengo** [mko/mowens.com-public#1]
You may not care about this, just thought I'd let you know.
https://github.com/mko/mowens.com-public/blob/master/lib/server.js#L555

It caught my eye because it's so similar to my address. I live two blocks east on Fulton!

**Comment by mko** [comment]
Yep. My address is public in a lot of places (WHOIS / domain registry, Microformats on my website, various public records, and more).

As far as it catching your eye, San Francisco tech is a remarkably small world. How'd you end up digging into my code to find the address data?
